jena-arq/src/main/java/org/apache/jena/sparql/lang/ParserSPARQL12Update.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.jena.sparql.lang;
+
+import java.io.Reader ;
+
+import org.apache.jena.atlas.logging.Log ;
+import org.apache.jena.query.QueryException ;
+import org.apache.jena.query.QueryParseException ;
+import org.apache.jena.shared.JenaException ;
+import org.apache.jena.sparql.core.Prologue;
+import org.apache.jena.sparql.lang.sparql_11.SPARQLParser11 ;
+import org.apache.jena.sparql.modify.UpdateSink ;
+import org.apache.jena.update.UpdateException ;
+
+public class ParserSPARQL12Update extends UpdateParser
+{
+    public ParserSPARQL12Update() {}
+
+    @Override
+    protected void executeParse(UpdateSink sink, Prologue prologue, Reader r) {
+        SPARQLParser11 parser = null ;
+        try {
+            parser = new SPARQLParser11(r);
```

Comment:
Shouldn't `ParserSPARQL12Update` be using `SPARQLParser12`?

Reply:
Or is it just for now until the SPARQL 1.2 is fully implemented?

Reply:
Thank you for catching that. I checked again, this time by deleting all the parsers! That reliably finds all uses in the codebase.